client/js/table-view.js

```diff
@@ -98,9 +115,22 @@ class TableView {
 		this.currentCellLocation = { col: col, row: row };
 		this.renderTableBody(); 
 		this.renderFormulaBar();
+		this.renderTableFooter();
 	}
 
-
+	calculateColumnSum(col) {
+		let sum = 0;
+		for(let row = 0; row < this.model.numRows; row++) {
+			if(this.model.getValue({col: col, row: row}) !== undefined) {
+				if(!isNaN(parseInt(this.model.getValue({col: col, row: row})))) {
```

Comment:
When you have nested "if" statements like this, you can combine them into a single "if" statement using the && operator.